src/index.js

```diff
@@ -1,28 +1,18 @@
-import db from './notifications/db';
-import Configuration from './notifications/config';
-import notificationController from './notifications/controller';
+import NotificationController from './notifications/controller';
 
 class Notification {
-  constructor(config) {
-    this.config = new Configuration(config);
-    this.configDatabase = this.config.database;
-    this.baseApi = this.config.baseApi;
+  constructor(database) {
+    this.notificationController = new NotificationController(database);
+
+    this.getAllUnread = this.notificationController.getAllUnread.bind(this.notificationController);
```

Comment:
I don't understand why you need to do this, can you explain? Also it seems that is not being used.